modules/core/src/shaderlib/lighting/lighting.glsl.js

```diff
@@ -19,51 +19,94 @@
 // THE SOFTWARE.
 
 // lighting
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
```

Comment:
Does this file implement `PhongMaterial` specifically? Should another shader be linked if another material is used? If so, should we name the shader file accordingly?